src/create-route-map.js

```diff
@@ -34,6 +34,21 @@ export function createRouteMap (
     }
   }
 
+  if (process.env.NODE_ENV === 'development') {
+    // warn if routes do not include leading slashes
+    const found = pathList
+    // check for missing leading slash
+      .filter(path => path && path.charAt(0) !== '*' && path.charAt(0) !== '/')
+    // split the path to get the root path part only
+      .map(path => path.split('/')[0])
+    // remove duplicates caused by split child paths
+      .filter((path, index, pathList) => pathList.indexOf(path) === index)
```

Comment:
I don't think this method is good as it will remove remaining path from `/some/longer/url` when it doesn't have children. So I think there should be some kind of mixing with the `pathMap` to make it display the same path that is displayed with the record, otherwise displaying the whole path seems simpler and easier to spot, even if it could lead to multiple paths when only one record is affected